pipeline/gnomad/gnomad_to_vcf.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+"""
+Description:
+    Takes in a gnomad table and converts it to vcf format.
+"""
+
+from __future__ import print_function, division
+import argparse
+import sys
+import os
+from collections import defaultdict
+import pyhgvs as hgvs
+import pyhgvs.utils as hgvs_utils
+from pygr.seqdb import SequenceFileDB
+import logging
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Convert database table to VCF format.')
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help='Input file for conversion.')
+    parser.add_argument('-a', '--inAnnot', default='/hive/groups/cgl/brca/phase1/data/resources/gnomadAnnotation',
```

Comment:
I wonder, whether it makes any sense to have default like those. I don't see any benefit except maybe for one developer working in a specific environment. It may be more useful to have paths valid in the docker image (but no default would be fine with me as well)